androidshared/src/main/java/org/odk/collect/androidshared/system/ContextUtils.kt

```diff
@@ -12,4 +16,36 @@ object ContextUtils {
         context.theme.resolveAttribute(resId, outValue, true)
         return outValue.data
     }
+
+    /** Renders a Drawable (such as a vector drawable) into a Bitmap.  */
+    @JvmStatic
+    fun getBitmap(context: Context, drawableId: Int): Bitmap? {
+        val drawable = ContextCompat.getDrawable(context, drawableId)
+        if (drawable != null) {
```

Comment:
Are you doing here anything special? If not you can probably use the `toBitmap` extension function:
`drawable.toBitmap()`

Reply:
I was just moving this around so hadn't given it a proper look, but I think you're right. It's only used for the OSM crosshairs anyway and switching it to using `toBitmap` seems to work!